packages/docusaurus-plugin-content-docs/src/__tests__/index.test.js

```diff
@@ -7,7 +7,7 @@
 
 import '@babel/polyfill';
 import path from 'path';
-import loadSetup from '../../../docusaurus/test/loadSetup';
+import loadSetup from '../../../docusaurus/lib/server/load/__tests__/loadSetup';
```

Comment:
Perhaps we should create some test utils. It's quite fragile for packages to cross module boundaries like this and import from other modules via relative paths.

Reply:
was thinking of that as well. Gonna do it soon, this PR is so big already. I haven't removed the logic on plugin-docs also. (doing it now)

Reply:
I didn't see the comment above about @docusaurus/test-utils because this PR has 100+ changed files 🤣

Reply:
Been working it for few hours too :| I'll double check again and again lol. ~3,000 lines of code deleted. gg